tools/tpch-tools/conf/doris-cluster.conf

```diff
@@ -26,4 +26,4 @@ export USER='root'
 # Doris password
 export PASSWORD=''
 # The database where TPC-H tables located
-export DB='tpch_test'
+export DB='tpch_test2'
```

Comment:
invalid change? or just change it to tpch without _test following other benchmark tools.

Reply:
this is not proper. I will remove it